### PR TITLE
*: add pprof flag to grpc proxy

### PR DESCRIPTION
**Pull request by xiang90**
Fix https://github.com/coreos/etcd/issues/7514

### CODE REVIEW

File: pkg/pprofutil/pprofutil.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 The etcd Authors
```
Comment (heyitsanthony) [inline pkg/pprofutil/pprofutil.go]: `pkg/debugutil`?

File: pkg/pprofutil/pprofutil.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+// Package pprofutil includes utility functions for pprof.
+package pprofutil
+
+import (
+	"net/http"
+	"net/http/pprof"
+)
+
+const HTTPPrefix = "/debug/pprof"
```
Comment (heyitsanthony) [inline pkg/pprofutil/pprofutil.go]: `HTTPPrefixPProf``?

File: pkg/pprofutil/pprofutil.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+// Handlers returns a map of pprof handlers keyed by the HTTP path.
+func Handlers() map[string]http.Handler {
```
Comment (heyitsanthony) [inline pkg/pprofutil/pprofutil.go]: `PProfHandlers`?

File: pkg/pprofutil/pprofutil.go (hunk 1 above)
Comment (xiang90) [inline pkg/pprofutil/pprofutil.go]: will change.

File: pkg/pprofutil/pprofutil.go (hunk 2 above)
Comment (xiang90) [inline pkg/pprofutil/pprofutil.go]: will change.

File: pkg/pprofutil/pprofutil.go (hunk 3 above)
Comment (xiang90) [inline pkg/pprofutil/pprofutil.go]: will change.

**Comment by xiang90** [comment]
@heyitsanthony fixed.

**Pull request merged**